devenv/dev-dashboards/datasource-testdata/new_features_in_v74.json

```diff
@@ -511,7 +511,6 @@
       },
```

Comment:
this file isn't tested because its schemaVersion is >30, but the `new_features_in_v8.json` is tested and it's got failures :)

Reply:
perfect -- I was mostly making a PR to see how the CI failed :)

And given the errors I would have figured it out :tada: